app/controllers/people_controller.rb

```diff
@@ -97,8 +96,7 @@ def new
 
   def edit
     @person = Person.find(params[:id])
-    #This is a quick hack to allow people to enter e-mails and phone numbers for person records already created
-    @person.channels.build (attributes = { status: "OK", usage: '1-All'}) if @person.channels.count < 2
+    @person.channels.build if @person.channels(true).empty?
```

Comment:
Yes. The contact information is powered by links via the cocoon gem. In order to get the first contact nested form to display (but blank), we need to build an empty channel for a person. It's basically providing the same functionality as the "quick hack" that Carlson(?) provided.